bin/main.rb

```diff
@@ -1,2 +1,117 @@
 #!/usr/bin/env ruby
-puts 'Hello World'
+# Implement TicTacToe Game Interface
+
+class TicTacToe
```

Comment:
Using a class is not necessary 😅

Reply:
Thanks for the Review @santiago-rodrig, It isn't the instruction that wasn't clear. It was a reviewer that insisted on building a fully functional game Logic @Daniel Laloush, please check other reviews at the top

Reply:
I'm aware of that, I already talked with him.